velox/docs/functions/spark/decimal.rst

```diff
@@ -2,47 +2,92 @@
 Decimal Operators
 =================
 
-When calculating the result precision and scale of arithmetic operators,
-the formulas follow Hive which is based on the SQL standard and MS SQL:
+The result precision and scale computation of arithmetic operators contains two stages.
+In the first stage, the formulas follow Hive which is based on the SQL standard and MS SQL.
+In the second stage, caps p at 38 and reduces scale or not according to the allowing precision loss mode.
+
+Decimal Precision and Scale Computation Formulas
+------------------------------------------------
+
+The SQL standard and MS SQL:
 
 https://cwiki.apache.org/confluence/download/attachments/27362075/Hive_Decimal_Precision_Scale_Support.pdf
 
 https://msdn.microsoft.com/en-us/library/ms190476.aspx
 
+Additionally, the computation of decimal division adapts to the allowing precision loss mode,
```

Comment:
> adapts to the allowing precision loss mode,

adapts to allow-precision-loss mode

> while the decimal addition, subtraction, and multiplication do not.

This sounds like results of +, -, * are not affected by allow-precision-loss mode. I don't believe this is the case. Is it?

Reply:
This is the first stage, only affect /